bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -125,6 +143,18 @@ protected void loadFavicons(String designPath) {
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    @JsonIgnore
+    public String[] getClientLibCategoriesJsBody() {
```

Comment:
See above: `getClientLibCategoriesJsBody` -> `getClientlibCategoriesBody` (lower case `L` and no `js`)

Reply:
* Still require `js` here
* For the `L` I'm could be convinced that we shouldn't be consistent with the existing API / property naming.